modules/era/lua_dynamis/globals/era_dynamis_spawning.lua

```diff
@@ -604,7 +604,7 @@ xi.dynamis.nonStandardDynamicSpawn = function(mobIndex, oMob, forceLink, zoneID,
             ["onMobSpawn"] = { function(mob) xi.dynamis.setNightmareStats(mob) end },
             ["onMobEngaged"] = { function(mob, target) end },
             ["onMobFight"] = { function(mob) end },
-            ["onMobRoam"] = { function(mob) xi.dynamis.mobOnRoam(mob) end },
+            ["onMobRoam"] = { function(mob) end },
```

Comment:
Why was this removed? This is needed for Dynamis Tav

Reply:
> Why was this removed? This is needed for Dynamis Tav

There is no such function, it is causing constant errors in log in CoP dynas